src/SDKs/_metadata/automation_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-01-18 21:18:29 UTC
+2018-05-10 20:38:47 UTC
```

Comment:
Please create a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj) and regenerate the code.

Reply:
Fixed.